packages/react/src/jsx/__tests__/index.test.tsx

```diff
@@ -129,4 +129,31 @@ describe('css prop', () => {
     expect(getByText('hello world')).toHaveCompiledCss('font-size', '20px');
     expect(getByText('hello world')).not.toHaveCompiledCss('font-size', '12px');
   });
+
+  it('should create css from object reference with true condition', () => {
+    const condition = true;
+    const base = { fontSize: 12 };
+    const { getByText } = render(<div css={condition && base}>hello world</div>);
+
+    expect(getByText('hello world')).toHaveCompiledCss('font-size', '12px');
+  });
+
+  it('should create css from object reference with false condition', () => {
+    const condition = false;
+    const base = { fontSize: 12 };
+    const { getByText } = render(<div css={condition && base}>hello world</div>);
+
+    expect(getByText('hello world')).not.toHaveCompiledCss('font-size', '12px');
```

Comment:
I want to use an empty expect value. But I don't know how to use it.

Reply:
hmmm that doesn't exist ATM

I think your assertion is okay though - it _is_ asserting the expected behaviour 😄